dev-client/android/app/src/main/java/com/devclient/MainActivity.java

```diff
@@ -1,12 +1,20 @@
 package com.devclient;
 
+import android.os.Bundle;
 import com.facebook.react.ReactActivity;
 import com.facebook.react.ReactActivityDelegate;
 import com.facebook.react.defaults.DefaultNewArchitectureEntryPoint;
 import com.facebook.react.defaults.DefaultReactActivityDelegate;
 
 public class MainActivity extends ReactActivity {
 
+
+  // https://reactnavigation.org/docs/getting-started/#installing-dependencies-into-a-bare-react-native-project
```

Comment:
It says a bit underneath:
> This change is required to avoid crashes related to View state being not persisted consistently across Activity restarts.
I'll add that to the comment as well.